torch/csrc/Module.cpp

```diff
@@ -1891,6 +1892,9 @@ PyObject* initModule() {
 #ifdef USE_MPS
   torch::mps::initModule(module);
 #endif
+#ifdef USE_VULKAN
```

Comment:
Can we do this ifdef insite the initModule?
This is a footgun where it becomes very easy to define APIs that exist or not on the python module depending on build flags.
This is a big issue for distributed that does that today as anyone that "import" your function in python will see their script crash in a weird way with another install of PyTorch. And catching these import errors is hard.

It is best if we could have always the same API but it throws an error if you try to use it.

Reply:
I tried to make this match MPS, which is directly above. I take it this is a case where it's too late to fix MPS but not Vulkan?

Reply:
Correct! :)